daffodil-cli/src/it/scala/org/apache/daffodil/blob/TestBlob.scala

```diff
@@ -23,31 +23,20 @@ import java.io.File
 import org.apache.daffodil.CLI.Util
 import net.sf.expectit.matcher.Matchers.contains
 import net.sf.expectit.matcher.Matchers.eof
+import net.sf.expectit.Expect
 
 class TestBlob {
 
-  val output1 = Util.getExpectedString("output1.txt")
-  val output1_nopretty = Util.getExpectedString("output1_nopretty.txt")
-  val output2 = Util.getExpectedString("output2.txt")
-  val output4 = Util.getExpectedString("output4.txt")
-  val output6 = Util.getExpectedString("output6.txt")
-  val output8 = Util.getExpectedString("output8.txt")
-  val output9 = Util.getExpectedString("output9.txt")
-  val output10 = Util.getExpectedString("output10.txt")
-  val output12 = Util.getExpectedString("output12.txt")
-
-  // Call md5sum with two different files and return if they match
-  /*def md5sum(shell: Expect, a: String, b: String): Boolean {
-    val cmdA = String.format("/usr/bin/md5sum %s | awk '{print $1}'", a)
-    shell.sendLine(cmd)
-    val resA = shell.expect("\n").toString
+  def md5sum(shell: Expect, blob_path: String, checksum: String): Unit = {
```

Comment:
MIght make more sense to put this in the Utils.scala file. Other CLI tests that aren't blobs might find this useful.